app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.where(author_id: params[:id])
+    @posts = Post.includes(:comments).where(author_id: params[:id])
```

Comment:
- [ ] Great work for solving the n+1 query problem by includes a reference to your post :+1: